packages/volto-blocks/src/components/blocks/IconsAndText/EditItem.tsx

```diff
@@ -0,0 +1,141 @@
+import type { ArrayElement } from '@plone/types';
+import { UniversalLink } from '@plone/volto/components';
+import { flattenToAppURL } from '@plone/volto/helpers';
+import type { IconsAndTextData } from '@redturtle/volto-blocks/components/blocks/IconsAndText/schema';
+import styles from '@redturtle/volto-blocks/components/blocks/IconsAndText/styles.module.scss';
+import { TextEditorWidget } from '@redturtle/volto-rt-slate';
+import cx from 'classnames';
+import { defineMessages, useIntl } from 'react-intl';
+
+type Props = {
+  data: ArrayElement<IconsAndTextData['columns']>;
+  focusOn: string;
+  setFocusOn: Function;
+  onChange: (id: string, field: string, value: unknown) => void;
+  selected: boolean;
+};
+
+export default function EditItem({
+  data,
+  focusOn,
+  setFocusOn,
+  onChange,
+  selected,
+}: Props) {
+  const intl = useIntl();
+  const icon = data.iconImage;
+
+  if (__SERVER__) {
+    return <div />;
+  }
+  return (
+    <div
+      className={cx(
+        'block-iconsandtext-column',
+        styles['column-block'],
+        styles['divider_' + data.dividerPosition],
+      )}
+    >
+      <div
+        className={cx(
+          'column-head',
+          styles['column-head'],
+          styles['header-text-' + data.headerTextPosition],
+        )}
+      >
+        {icon && (
+          <div
+            className={cx(
+              'column_icon size_',
+              styles['column_icon'],
+              styles[`size_${data.iconSize}`],
+            )}
+          >
+            <img
+              src={flattenToAppURL(icon) + '/@@images/image/teaser'}
+              alt=""
+              role="presentation"
+              aria-hidden="true"
+            />
```

Comment:
Using the Image component from Volto would be better here for future compatibility. See the View component of the Text7 block for a working example.